subsys/bluetooth/controller/Kconfig

```diff
@@ -293,6 +293,22 @@ config BT_CTLR_RL_SIZE
 	  Set the size of the Resolving List for LE Controller-based Privacy.
 	  On nRF5x-based controllers, the hardware imposes a limit of 8 devices.
 
+config BT_CTLR_SW_DEFERRED_PRIVACY
+	bool "LE Controller-based Software Privacy"
+	depends on BT_CTLR_PRIVACY
+	default n
+	help
+	  Enable support for software based deferred privacy calculations.
+
+config BT_CTLR_KU_RL_SIZE
```

Comment:
What does `KU` stand for?

Reply:
Known Unknown (RPAs). If you don't have crypto HW acceleration, resolving an RPA can be time consuming. Keeping a list of known unresolvable RPAs will keep us from recalculating the same thing.

Reply:
Requested changes implemented